Home-exercises/src/io/codelex/arithmetic/practice/Exercise1.java

```diff
@@ -0,0 +1,22 @@
+package io.codelex.arithmetic.practice;
+
+import java.util.Scanner;
+
+public class Exercise1 {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.print("Please input first number: ");
+        int a = in.nextInt();
+        System.out.print("Please input second number: ");
+        int b = in.nextInt();
+        boolean f = false;
+        if (a == 15 | b == 15) {
```

Comment:
Bitwise | operatoru neizmantojam, liec parasto or ||. Tas pats arī zemāk.

Kā arī labāk šo būtu uzrakstīt vienā izteiksmē nevis ar IF blokiem, piemēram, šādi:
boolean result = a == 15 || b == 15 || a - b == 15 || b - a == 15 || a + b == 15;

<= manu kodu vēl var uzlabot izmantojot absolūto vērtību, lai nav jāpārbauda gan a - b gan b - a.